tests/spec/core/issues-notes-spec.js

```diff
@@ -160,6 +160,32 @@ describe("Core — Issues and Notes", function() {
     );
   });
 
+  it("removes closed issues from spec", async () => {
+    const githubConfig = {
+      github: "https://github.com/mock-company/mock-repository",
+      githubAPI: `${window.location.origin}/tests/data/mock-github-api-v3-response.json`
```

Comment:
this may end up being incompatible with:
https://github.com/w3c/respec/pull/1560/files#diff-0cfbbabe5bc2a8772222d1152309dd3eR144

Here we probably want: `${window.location.origin}/tests/data/`

And we should resolve to: `${window.location.origin}/tests/data/issues/1540`

"1540" should just be a the json file. Alternatively, you might be able to just add "?testing=", which should also work: 

```js
${window.location.origin}/tests/data/mock-github-api-v3-issue-response.json?testing=
```